2019-03-06 17:26:28,859-selenium.webdriver.remote.remote_connection-INFO-Could not connect to port 4723 on host 127.0.0.1
2019-03-06 17:26:28,859-selenium.webdriver.remote.remote_connection-INFO-Could not connect to port 4723 on host 127.0.0.1
2019-03-06 17:26:28,859-selenium.webdriver.remote.remote_connection-INFO-Could not get IP address for host: 127.0.0.1
2019-03-06 17:26:28,859-selenium.webdriver.remote.remote_connection-INFO-Could not get IP address for host: 127.0.0.1
2019-03-06 17:26:29,889-urllib3.connectionpool-WARNING-Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000024B2BB0CF28>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。',)': /wd/hub/session
2019-03-06 17:26:29,889-urllib3.connectionpool-WARNING-Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000024B2BB0CF28>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。',)': /wd/hub/session
2019-03-06 17:26:30,899-urllib3.connectionpool-WARNING-Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000024B2BB2F0B8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。',)': /wd/hub/session
2019-03-06 17:26:30,899-urllib3.connectionpool-WARNING-Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000024B2BB2F0B8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。',)': /wd/hub/session
2019-03-06 17:26:31,902-urllib3.connectionpool-WARNING-Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000024B2BB2F1D0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。',)': /wd/hub/session
2019-03-06 17:26:31,902-urllib3.connectionpool-WARNING-Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000024B2BB2F1D0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。',)': /wd/hub/session
2019-03-06 17:26:49,505-selenium.webdriver.remote.remote_connection-INFO-Could not connect to port 4723 on host 127.0.0.1
2019-03-06 17:26:49,505-selenium.webdriver.remote.remote_connection-INFO-Could not connect to port 4723 on host 127.0.0.1
2019-03-06 17:26:49,505-selenium.webdriver.remote.remote_connection-INFO-Could not get IP address for host: 127.0.0.1
2019-03-06 17:26:49,505-selenium.webdriver.remote.remote_connection-INFO-Could not get IP address for host: 127.0.0.1
2019-03-06 17:26:50,508-urllib3.connectionpool-WARNING-Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000265633DDEB8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。',)': /wd/hub/session
2019-03-06 17:26:50,508-urllib3.connectionpool-WARNING-Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000265633DDEB8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。',)': /wd/hub/session
2019-03-06 17:26:51,509-urllib3.connectionpool-WARNING-Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000265633DDFD0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。',)': /wd/hub/session
2019-03-06 17:26:51,509-urllib3.connectionpool-WARNING-Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000265633DDFD0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。',)': /wd/hub/session
2019-03-06 17:26:52,513-urllib3.connectionpool-WARNING-Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000265633FF160>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。',)': /wd/hub/session
2019-03-06 17:26:52,513-urllib3.connectionpool-WARNING-Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000265633FF160>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。',)': /wd/hub/session
